### PR TITLE
Put failing tests behind ActiveIssue

### PR DESCRIPTION
**Pull request by MichalStrehovsky**

### CODE REVIEW

File: src/coreclr/nativeaot/System.Private.CoreLib/src/System/Enum.CoreRT.cs [hunk 1]
```diff
@@ -222,7 +222,7 @@ public static string[] GetNames(Type enumType)
                 return enumType.GetEnumNames();
 
             if (!enumType.IsEnum)
-                throw new ArgumentException(SR.Arg_MustBeEnum);
+                throw new ArgumentException(SR.Arg_MustBeEnum, nameof(enumType));
```
Comment (MichalStrehovsky) [inline src/coreclr/nativeaot/System.Private.CoreLib/src/System/Enum.CoreRT.cs:225]: Oh well. My local repo is a mess so I would prefer this is just taken for the ride.

File: src/libraries/System.Runtime/tests/System/Type/TypeTests.cs [hunk 2]
```diff
@@ -527,6 +527,7 @@ public void GetTypeByName_Invalid(string typeName, Type expectedException, bool
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtimelab/issues/155" /* NativeAot */)]
         public void GetTypeByName_InvokeViaReflection_Success()
```
Comment (jkotas) [inline src/libraries/System.Runtime/tests/System/Type/TypeTests.cs:531]: What is the problem with this one?

**Review by jkotas — APPROVED** [review]
Thanks!

File: src/libraries/System.Runtime/tests/System/Type/TypeTests.cs (hunk 2 above)
Comment (MichalStrehovsky) [reply src/libraries/System.Runtime/tests/System/Type/TypeTests.cs:531]: I didn't debug it, but the type is not assembly qualified so we're supposed to look in the assembly that is calling the API. I assume CoreCLR knows the calling assembly even when reflection is involved. I figure this was needed for CAS, but for native AOT I assume we can put this in the "never going to happen" bucket.

**Pull request merged**